vendor/engines/sanger_sequencing/app/services/sanger_sequencing/sample_result_file_saver.rb

```diff
@@ -23,7 +23,9 @@ def save
                                    file_type: "sample_result", created_by: current_user.id,
                                    order_detail: sample.submission.order_detail,
                                    product_id: sample.submission.order_detail.product_id)
-      stored_file.save
+      success = stored_file.save
+      trigger_notification(stored_file) if success
+      success
```

Comment:
Yeah, I thought about that, but I didn't want to need to rely on the return value of `trigger_notification`, or rather, didn't want to rely on the return value of `ResultsFileNotifier.new(stored_file).notify`, which returns nil for the case where the feature is turned off or it doesn't actually send an email because it's already sent one recently.